### ISSUE TITLE
Linux command line R with bigrquery returns "Error:"

### ISSUE THREAD

**Issue by matthewtgross** [rstats-db/bigrquery#86]
Linux command line R with bigrquery returns "Error:", as in error with no details.  Does anyone have any idea what I might be doing wrong?  I just ran an update for all installed packages.  Could this be an issue with a new package?

**Comment by craigcitro** [comment]
what code were you trying to run?

**Comment by matthewtgross** [comment]
> sql <- "select current_date();"
> current_date <- query_exec(sql, project, max_pages = Inf)
Error:

**Comment by fabianbruckschen** [comment]
I experienced the exact same issue after updating packages on my linux machine.
httr was one of the updated packages

**Comment by davidFMP** [comment]
bigrquery is incompatible with httr 1.1.0. The quick fix is to install stringr, stringi, an digest independently and then httr 1.0.0 from the CRAN archive
https://cran.r-project.org/src/contrib/Archive/httr/

Then, install bigrquery without dependencies.

**Comment by fabianbruckschen** [comment]
Don't mind me asking, but what is the sense of installing stringr/stringi here? Can't you just revert to it like this:
install.packages('https://cran.r-project.org/src/contrib/Archive/httr/httr_1.0.0.tar.gz', repos = NULL, type = 'source')

**Comment by davidFMP** [comment]
I seem to have removed these packages somewhere along the line, when trying to figure out where the bug is, so needed to reinstall them. Otherwise, yes you can just roll back

**Comment by krlmlr** [comment]
I don't experience any problems locally (Ubuntu 14.10):

```
> sessionInfo()
R version 3.2.3 (2015-12-10)
Platform: x86_64-pc-linux-gnu (64-bit)
Running under: Ubuntu 15.10

locale:
 [1] LC_CTYPE=en_US.UTF-8       LC_NUMERIC=C               LC_TIME=de_CH.UTF-8        LC_COLLATE=en_US.UTF-8    
 [5] LC_MONETARY=de_CH.UTF-8    LC_MESSAGES=en_US.UTF-8    LC_PAPER=de_CH.UTF-8       LC_NAME=C                 
 [9] LC_ADDRESS=C               LC_TELEPHONE=C             LC_MEASUREMENT=de_CH.UTF-8 LC_IDENTIFICATION=C       

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
[1] bigrquery_0.1.0.9000 testthat_0.11.0.9000 magrittr_1.5        

loaded via a namespace (and not attached):
 [1] Rcpp_0.12.3          roxygen2_5.0.1       dplyr_0.4.3          digest_0.6.9         crayon_1.3.1         withr_1.0.1         
 [7] assertthat_0.1       R6_2.1.2             jsonlite_0.9.19      DBI_0.3.1.9008       DBItest_1.0-6        httr_1.1.0          
[13] stringi_1.0-1        curl_0.9.5           devtools_1.10.0.9000 tools_3.2.3          stringr_1.0.0        parallel_3.2.3      
[19] memoise_1.0.0        openssl_0.9.1       
```

**Comment by demorenoc** [comment]
Just to document: I ran into this same issue and after some debugging I found it is indeed related to the version change on httr from 1.0.0 to 1.1.0, as is noted in this commit: https://github.com/hadley/httr/commit/a7deaa1ad98158ae8abc3f9dfb843a5554fd80fa. So yes this is solved in the development version of bigrquery (0.1.0.9000) or alternatively rollingback to httr 1.0.0. Hopefully the CRAN version gets updated soon.
